api.md

```diff
@@ -650,7 +650,7 @@ add:
       {:ok, upload_response_body} =
         image.path
         |> ExAws.S3.Upload.stream_file()
-        |> ExAws.S3.upload(@original_bucket, file_name,
+        |> ExAws.S3.upload(Application.get_env(:ex_aws, :original_bucket), file_name,
```

Comment:
Definitely not a fan of having these `Application.get_env` calls in the function invocation like this.

Reply:
If this is the case, I guess we'll have to live with the warning and revert these changes so they use `module_attributes`

Reply:
Nah, there's a midway. Gimme a minute ... 🧑‍💻 ⌛